ci-operator/config/openshift/oauth-apiserver/openshift-oauth-apiserver-release-4.5.yaml

```diff
@@ -8,10 +8,15 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.12
+    tag: golang-1.13
```

Comment:
you don't have to edit these branches - all above master (not cut yet) are fast forwarded - it also limits the CI to branches that you care about - 4.5 is likely flaky as it's being setup just to test the configs